api/identityfile/identityfile.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package client
+// Package identityfile provides the IdentityFile struct and its methods.
```

Comment:
nit: prefer to describe the purpose of the package rather than its contents
```suggestion
// Package identityfile implements parsing and serialization of Teleport identity files.
```